src/Codec/Xlsx/Formatted.hs

```diff
@@ -236,6 +238,25 @@ formatted cs styleSheet =
         , formattedMerges     = reverse (finalSt ^. formattingMerges)
         }
 
+formatSingleWorksheet :: Map (Int, Int) FormattedCell -> StyleSheet -> (Worksheet, StyleSheet)
+formatSingleWorksheet fcs styleSheet = (ws, styleSheet')
```

Comment:
this function looks to contain a lot of `formatted` - please either reuse `formatted` or you could introduce some common helper function though I don't see much sense in that